### PR TITLE
Add .gitattributes to make spotlessApply work on Windows

### PR DESCRIPTION
**Pull request by TWiStErRob**
Original behavior: `gradlew spotlessApply` -> all files changed

Steps:
 * look up why -> https://github.com/diffplug/spotless/issues/39
 * add .gitattributes
 * reset working copy -> https://stackoverflow.com/q/48642692/253468

New behavior: If I run spotless again, it changes nothing, unless there's actually something wrong.

Please someone on Mac/linux check out the branch test this (on an existing and a new clone):
 * change a file in `paparazzi/paparazzi-gradle-plugin/src/main/kotlin`
 * `git status` (observe changed file)
 * run `./gradlew -p paparazzi spotlessApply`
 * `git status` (observe no changed files)

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Source code
+*.kt            text eol=lf
```
Comment (JakeWharton) [inline .gitattributes:2]: Make the default LF and only add exemptions. Like https://github.com/square/retrofit/blob/master/.gitattributes. Less whack-a-mole.

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+*.java          text eol=lf
+*.xml           text eol=lf
+
+# Gradle files
+*.gradle        text eol=lf
+*.properties    text eol=lf
+*.bat           text eol=crlf
```
Comment (TWiStErRob) [inline .gitattributes:9]: Something is fishy still, even after resetting gradlew.bat keeps changing, any ideas welcome.

I tried autocrlf=true and false, and then reset, but no luck.

File: build.gradle [hunk 3]
```diff
@@ -68,7 +68,3 @@ subprojects {
 tasks.register("clean", Delete).configure {
   delete rootProject.buildDir
 }
-
-tasks.named("wrapper").configure {
-  distributionType = Wrapper.DistributionType.ALL
```
Comment (JakeWharton) [inline build.gradle:73]: Big fan. But should be in a separate PR.

Comment (TWiStErRob) [reply build.gradle:73]: k

File: .gitattributes (hunk 2 above)
Comment (TWiStErRob) [reply .gitattributes]: Re-cloned and all good, but I was able to reproduce it with the code here: https://stackoverflow.com/q/48642692/253468

This however fixes it: https://stackoverflow.com/a/19588910/253468

Right now it's above my git abilities, but it should be fine considering retrofit uses the same file.

**Review by jrodbx — APPROVED** [review]

File: .gitattributes (hunk 2 above)
Comment (TWiStErRob) [reply .gitattributes]: It was solved in: https://github.com/cashapp/paparazzi/pull/721
